hpe3parclient/client.py

```diff
@@ -1895,6 +1912,36 @@ def _getProtocolPorts(self, protocol, state=None):
 
         return return_ports
 
+    def _cloneISCSIPorts(self, real_ports, vlan_ports):
+        cloned_ports = []
+        for port in vlan_ports:
+            matching_ports = [
+                x for x in real_ports['members']
+                if (x['protocol'] == self.PORT_PROTO_ISCSI and
+                    x['iSCSIPortInfo']['vlan'] == 1 and
+                    x['portPos'] == port['portPos'])
+            ]
+
+            # should only be one
+            if len(matching_ports) > 1:
+                err = ("Found {} matching ports for vlan tagged iSCSI port "
+                       "{}.  There should only be one.")
+                raise exceptions.\
+                    NoUniqueMatch(err.format(len(matching_ports), port))
+
+            if len(matching_ports) == 1:
+                new_port = copy.deepcopy(matching_ports[0])
+                new_port.update(port)
+                cloned_ports.append(new_port)
+
+        return cloned_ports
+
+    def _getISCSIVlans(self):
+        self.ssh.open()
+        iscsi_vlan_data = self.ssh.run(['showport', '-iscsivlans'])
+        self.ssh.close()
```

Comment:
Instead of this, we can directly do 
return self._run(['showport', '-iscsivlans'])